### DISCUSSION TITLE
how to download plots?

### DISCUSSION THREAD

**Discussion by kexinliu0001** [nextpyp/.github#33] [Q&A] [answered]
hi,

I want to down load the plots such as CTF FIT, Est. Resolution, etc.
<img width="455" alt="image" src="https://github.com/nextpyp/.github/assets/110724494/c7df77a5-5313-43fc-8dc8-11fd871d4dd1">

but what I got is txt files:
<img width="646" alt="image" src="https://github.com/nextpyp/.github/assets/110724494/ce013e34-eca0-4fb5-b977-3f83742d99b9">

how can I get the plots?
or  are they already in somewhere of my folder (I found the images/photos but cannot find the plots)?

**Comment by cuchaz** [comment, marked as answer]
Looks like that's an [SVG](https://en.wikipedia.org/wiki/SVG) file? Try renaming it to `.svg` and see if your image viewer can open it.

The plot download process should create the `.svg` file for you. Is the file coming out as a `.txt` file instead?

**Reply by kexinliu0001** [reply]
yes! thank you!

the file came out as .txt file.
I renamed it and my viewer can open it.

**Reply by cuchaz** [reply]
Ok, I'm glad you got it working!

We'll look into why the file was incorrectly created as a `.txt` file.
